templates/xr-huaweivr/template/app/src/com/cocos/game/AppActivity.java

```diff
@@ -0,0 +1,125 @@
+/****************************************************************************
+Copyright (c) 2015-2016 Chukong Technologies Inc.
+Copyright (c) 2017-2018 Xiamen Yaji Software Co., Ltd.
```

Comment:
error time

Reply:
This is the code copied from the Android template. Do you also need to modify the time?